docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -598,17 +604,23 @@ To process the received command on the server side, follow these steps:
    ret = mc_server_set_playback_action_cmd_received_cb(g_mc_server, playback_action_cmd_received_cb, NULL);
    ```
 
-   - To register a callback for playback state change commands, use `mc_server_set_playback_action_cmd_received_cb()`.
-   - To register a callback for playback position change commands, use `mc_server_set_playback_position_cmd_received_cb()`.
-   - To register a callback for shuffle mode change commands, use `mc_server_set_shuffle_mode_cmd_received_cb()`.
-   - To register a callback for repeat mode change commands, use `mc_server_set_repeat_mode_cmd_received_cb()`.
-   - To register a callback for played item, playback state, and playback position change commands in playlist, use  `mc_server_set_playlist_cmd_received_cb()`.
-   - To register a callback for a custom command, use `mc_server_set_custom_cmd_received_cb()`.
-
+   - `mc_server_set_playback_action_cmd_received_cb()`: For playback state change commands.
+   - `mc_server_set_playback_position_cmd_received_cb()`: For playback position change commands.
+   - `mc_server_set_shuffle_mode_cmd_received_cb()`: For shuffle mode change commands.
+   - `mc_server_set_repeat_mode_cmd_received_cb()`: For repeat mode change commands.
+   - `mc_server_set_playlist_cmd_received_cb()`: For played item, playback state, and playback position change commands in playlist.
+   - `mc_server_set_custom_cmd_received_cb()`: For a custom command.
+   
+   > Since Tizen 5.5, following APIs are also supported.
```

Comment:
Just a small comment, please end the line with a colon(:) as the word 'following' is used in the sentence.

Since Tizen 5.5, the following APIs are also supported:

Reply:
Also, please remove \> in front of the line.

Reply:
Done